### PR TITLE
Image

### PR DESCRIPTION
**Pull request by pngwn**
# Description

This (i think) finalises the image related input components that form part of the svelte migration.

I had to tweak some settings to get things working.

- adds image editor
- add sketch tool
- adds webcam image capture

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: frontend-svelte/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public --no-clear"
+    "start": "sirv public --no-clear --port=3000"
```
Comment (pngwn) [inline frontend-svelte/package.json:8]: The default port (5000) is reserved on the latest version of macos, so i had to change this.

File: frontend-svelte/public/index.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <link rel='stylesheet' href='/build/bundle.css'>
+  <link rel="stylesheet" href="./global.css"
```
Comment (pngwn) [inline frontend-svelte/public/index.html:8]: Needed to add some global styles for tui editor and cropper js.

File: frontend-svelte/rollup.config.js [hunk 3]
```diff
@@ -40,13 +40,14 @@ export default {
...
-			process: JSON.stringify({
-				env: {
-					isProd: production,
-				}
-			}),
+			// process: JSON.stringify({
+			// 	env: {
+			// 		isProd: production,
+			// 	}
+			// }),
```
Comment (pngwn) [inline frontend-svelte/rollup.config.js:50]: This was replacing `process` including functions called `process` in any scope. Do we need to replace the whole process object? Where is `process.env.isProd` used? I couldn't find it when i had a quick look.

File: gradio.egg-info/PKG-INFO [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
 Author: Abubakar Abid, Ali Abid, Ali Abdalla, Dawood Khan, Ahsen Khaliq
 Author-email: team@gradio.app
 License: Apache License 2.0
-Description: UNKNOWN
 Keywords: machine learning,visualization,reproducibility
 Platform: UNKNOWN
+License-File: LICENSE
+
+UNKNOWN
+
```
Comment (pngwn) [inline gradio.egg-info/PKG-INFO:14]: I do not know what these changes are

File: gradio.egg-info/requires.txt [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+markdown2
 pycryptodome
-pydub
 requests
+paramiko
+analytics-python
+Flask>=1.1.1
+Flask-Cors>=3.0.8
+flask-cachebuster
+Flask-Login
```
Comment (pngwn) [inline gradio.egg-info/requires.txt:15]: Nor these.

File: gradio.egg-info/PKG-INFO (hunk 4 above)
Comment (FarukOzderim) [reply gradio.egg-info/PKG-INFO:14]: @pngwn the egg file is created when you install the gradio package, it is probably coming with setup-tools;

I think you can remove egg related files from the PR

[Ref](https://www.python101.pythonlibrary.org/chapter38_eggs.html)

Comment (pngwn) [reply gradio.egg-info/PKG-INFO:14]: Thanks!

File: frontend-svelte/package-lock.json [hunk 6]
```diff
@@ -1,4286 +1,8 @@
 {
```
Comment (pngwn) [inline frontend-svelte/package-lock.json]: This is a bit of a mess, I'll switch my npm version and sort this out.

**Pull request merged**